### PR TITLE
Be06 get article comments

### PR DESCRIPTION
**Pull request by tartebabba**
Added test for empty comments and article_id not existing.

@Saima: Please review all three tests for the get articles comments, as the most recent PR will only show the modified last two

### CODE REVIEW

File: __tests__/integrations.test.js [hunk 1]
```diff
@@ -185,12 +185,22 @@ describe('Articles', () => {
...
     });
-    test('GET 404: Endpoint returns error message when article has no comments', () => {
+    test('GET 200: Endpoint returns an empty array when article has no comments', () => {
       return request(app)
         .get('/api/articles/2/comments')
-        .expect(404)
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          const hasComments = comments.length;
+          expect(hasComments).toBeFalsy();
```
Comment (nc-richard-neat) [inline __tests__/integrations.test.js:194]: Might have been more readable to just assert an empty array here.

File: __tests__/integrations.test.js [hunk 2]
```diff
@@ -185,12 +185,22 @@ describe('Articles', () => {
...
+        });
+    });
+    test('GET 400: Endpoint returns error message when article does not exist', () => {
+      return request(app)
+        .get('/api/articles/200/comments')
+        .expect(400)
```
Comment (nc-richard-neat) [inline __tests__/integrations.test.js:200]: This would be a 400, it's not that its a bad request it's that we can't find that article.

**Review by nc-richard-neat — COMMENTED** [review]
Well done adding in this extra test, I've only commented on the new code so please check any previous feedback re old code 👍

File: __tests__/integrations.test.js (hunk 1 above)
Comment (tartebabba) [reply __tests__/integrations.test.js:194]: playing around with different matchers as don't want to just use the same ones all the time

Comment (nc-richard-neat) [reply __tests__/integrations.test.js:194]: Fine to have a go, but if the end result is not as readable as a solution we have used before I would go with the readable option at all times. The goal is always good, readable code.

**Pull request merged**